packages/gatsby/src/redux/actions.js

```diff
@@ -1142,6 +1142,65 @@ actions.addThirdPartySchema = (
   }
 }
 
+/**
+ *
+ * Report that a query has been extracted from a component. Used by
+ * query-compilier.js.
+ *
+ * @param {Object} $0
```

Comment:
I want to use actions more for internal communication but don't want them all to show up here (https://www.gatsbyjs.org/docs/actions/)

Thinking we could start marking actions as `private` to get around this?

Reply:
I'm all for using actions more (and less event emitting). Can we move them to the `actions` folder that already exists (but only has the page dependencies stuff), and export a subset that can be used in node APIs?

Reply:
I like just using pick :)